inc/az_ulib_ipc_api.h

```diff
@@ -454,6 +454,52 @@ AZ_NODISCARD az_result az_ulib_ipc_call_with_str(
     az_span model_in_span,
     az_span* model_out_span);
 
+/**
+ * @brief   Split the provided method full name.
+ *
+ * This function splits the provided method name between device, package, interface and capability
+ * using the following rules
+ *
+ * 1. The method full name shall be split by `.`.
+ * 2. The names and versions shall be extract from right to left.
+ * 3. The expected method full name shall looks like:
+ *      <device_name>.<package_name>.<package_version>.<interface_name>.<interface_version>.<capability_name>
+ * 4. The interface_name and interface_version are mandatory.
+ * 5. If a name is not provided, this function shall return #AZ_SPAN_EMPTY.
```

Comment:
can you specify "interface name" here?

Reply:
It is not the interface, it is for other names, I'll specify it.